core/server/master/src/main/java/alluxio/master/meta/AlluxioMasterRestServiceHandler.java

```diff
@@ -850,20 +849,22 @@ public Response getWebUIMetrics() {
       MasterWebUIMetrics response = new MasterWebUIMetrics();
 
       MetricRegistry mr = MetricsSystem.METRIC_REGISTRY;
+      SortedMap<String, Gauge> gauges = mr.getGauges();
+      SortedMap<String, Counter> counters = mr.getCounters();
```

Comment:
Every `MetricRegistry.getGauges` and `getCounters()` will do a traverse of all metrics, make a copy and return. So it will be better to reduce the call to these two methods.